application/src/main/java/org/togetherjava/tjbot/commands/help/HelpThreadActivityUpdater.java

```diff
@@ -121,6 +108,22 @@ private static RestAction<HelpSystemHelper.ThreadActivity> determineActivity(
         });
     }
 
+    private static RestAction<List<Message>> getRelevantHistory(MessageChannel channel) {
+        Long mostRecentMessageId =
+                manuallyResetChannelActivityCache.getIfPresent(channel.getIdLong());
+        RestAction<List<Message>> restActionMessages;
+
+        if (mostRecentMessageId != null) {
+            MessageHistory.MessageRetrieveAction historyAfter =
+                    channel.getHistoryAfter(mostRecentMessageId, ACTIVITY_DETERMINE_MESSAGE_LIMIT);
+            restActionMessages = historyAfter.map(MessageHistory::getRetrievedHistory);
+        } else {
+            restActionMessages =
+                    channel.getHistory().retrievePast(ACTIVITY_DETERMINE_MESSAGE_LIMIT);
+        }
+        return restActionMessages;
```

Comment:
might be readable enough with a ternary:
```java
return mostRecentMessageId != null
  ? channel.getHistoryAfter(...).map(...);
  : channel.getHistory().retrievePast(...);
```

either way, id get rid of the variable and return directly.

Reply:
this

Reply:
done.